### PR TITLE
Add optional emit for active support notifications

### PR DESCRIPTION
**Pull request by jcalvert**
This pull request allows a user of the plugin to specify a regular expression to subscribe to `ActiveSupport::Notifications` and then re-emit those notifications for processing downstream, such as with with `<match>` directives in the Fluentd configuration. This is useful because the Ruby Kafka gem [already supports ActiveSupport notifications](https://github.com/zendesk/ruby-kafka#instrumentation) allowing users to gather statistics and information.

### CODE REVIEW

File: lib/fluent/plugin/out_kafka.rb [hunk 1]
```diff
@@ -60,6 +60,8 @@ class Fluent::KafkaOutput < Fluent::Output
   config_param :max_buffer_bytesize, :integer, :default => nil,
                :desc => "Maximum size in bytes to be buffered."
 
+  config_param :active_support_notification_regex, :string, :default => nil
```
Comment (repeatedly) [inline lib/fluent/plugin/out_kafka.rb]: Add `:desc` parameter

**Comment by jcalvert** [comment]
@repeatedly Updated with description and using the router to more gracefully `emit` the message instead of just using `log`. Thank you!

**Comment by jcalvert** [comment]
bump please

**Comment by repeatedly** [comment]
Sorry. Just released v0.5.6.

**Comment by jcalvert** [comment]
Thank you! 🙇

**Pull request merged**